web3-storage/web3.storage
# feat: logins of already-existing users result in the user's customer contact info being updated (e.g. in stripe.com dashboard)

Motivation:
* business wants the stripe dashboard to be useful as a CRM of sorts, i.e. have pretty up to date contact info on the customer records there
  * before this PR, we only write that contact info for new user signups. This PR makes logins of previously-existing-users also result in an update of the customer contact info in stripe.
* https://www.notion.so/on-user-login-update-stripe-com-customer-with-latest-name-email-fed8d970688f4bfd8281dc6cc31581bc

Todo
* [x] write main business logic, tests that work against mock (not against stripe apis)
* [x] unit test for StripeCustomerService updateContact
* [x] integration test for StripeCustomerService updateContact
* [x] manual test login locally after stripe, ensure no errors

Challenges
* testing everything locally against an old user is hard because localdev always has an emptyish database of users.

How to Test on staging/production
* log in with email should work without error
* log in with github should work without error
* in prod, @gobengo can test with his oldish user and ensure that re-logging in will update the corresponding name/email of customer in stripe
